certbot-nginx/certbot_nginx/tests/configurator_test.py

```diff
@@ -447,7 +447,10 @@ def test_get_snakeoil_paths(self):
     def test_redirect_enhance(self):
         # Test that we successfully add a redirect when there is
         # a listen directive
-        expected = ['return', '301', 'https://$host$request_uri']
+        from certbot_nginx.configurator import _redirect_block_for_domain
+        from certbot_nginx.nginxparser import UnspacedList
```

Comment:
nit: Since we're not testing this functionality here, you should move this import to the top of the file.

Reply:
I don't really care if you fix this or not.